### ISSUE TITLE
Weird Dry Springs

### ISSUE THREAD

**Issue by DougLefelhocz** [jasonrohrer/OneLife#508]
Playing on server12 I found a southward gradient dry spring.  I go south and find another southward gradient dry spring.  I go south again and find a natural spring (not a well or pump).  What I believe happened was that the well reset due to the weekly reseting of the area to save space.  But, the dry springs did not reset.

**Comment by DougLefelhocz** [comment]
I saw this area again.  I decided to try and die there, but ran out of time looking for it.  In the next life I looked a little more carefully, found it, and made a fence there, and parked my horse there and died by the natural spring south of the two dry springs with stakes.  I named myself Eve Looker, so we can get absolute coordinates where this occurred.  No need to see the family tree on this one, and my daughter read someone else's note, which said something which doesn't seem like her to say.
